pinot-query-runtime/src/main/java/org/apache/pinot/query/service/dispatch/DispatchClient.java

```diff
@@ -41,7 +45,17 @@ class DispatchClient {
   private final PinotQueryWorkerGrpc.PinotQueryWorkerStub _dispatchStub;
 
   public DispatchClient(String host, int port) {
-    _channel = ManagedChannelBuilder.forAddress(host, port).usePlaintext().build();
+    this(host, port, new GrpcConfig(Collections.emptyMap()));
+  }
+
+  public DispatchClient(String host, int port, GrpcConfig grpcConfig) {
```

Comment:
This constructor is only used in the other constructor to create non-tls DispatchClient. Does it mean that if gRPC TLS is enabled on the server side, then DispatchClient will always fail?